src/fondue/Fondue.js

```diff
@@ -798,3 +942,76 @@ export default class Fondue {
 		return allGlyphs;
 	}
 }
+
+const createType6Summary = (feature, randomize) => {
+	let allInputs = [];
+	let allBacktracks = [];
+	let allLookaheads = [];
+
+	const limit = 10; // Summary will be limited to a max of limit³ (e.g. 20*20*20 = 8000)
+
+	const shuffleArray = (array) => {
+		for (let i = array.length - 1; i > 0; i--) {
+			const j = Math.floor(Math.random() * (i + 1));
+			[array[i], array[j]] = [array[j], array[i]];
+		}
+		return array;
+	};
+
+	// Create some kind of "all backtracks" or "all lookaheads"
+	for (const lookup of feature["lookups"]) {
+		if (lookup.type !== 6) continue;
+
+		// Create all possible combinations of input, backtrack and lookahead
+		for (const key in Object.entries(lookup["input"])) {
+			allInputs = [...new Set(allInputs.concat(lookup["input"][key]))];
```

Comment:
Given that you're creating Sets here again, it's probably going to be a lot more efficient if you change the whole lookup 6 type code to use Sets instead of Arrays. Certainly if the domain is such that these are actually sets and not arrays. WDYT?